BlueprintUI/Sources/Layout/GridRow.swift

```diff
@@ -0,0 +1,191 @@
+import UIKit
+
+/// Like `Row`, `GridRow` displays a list of items in a linear horizontal layout. Unlike `Row`, `GridRow` provides
+/// convenience for describing columnar layout.
+///
+/// Horizontally, `GridRow` children are stretched to fill the available space. Vertically, children are justified to
+/// fill the space of the tallest child.
+///
+/// Children may be sized proportionally or absolutely. Proportionally-sized children are granted a proportion of
+/// the total layout space **after** absolutely-sized children and margins have been subtracted.
+///
+/// ## Example:
+///
+/// ```
+/// GridRow { row in
+///   row.spacing = 8.0
+///   row.add(child: .proportional(name, 0.75))
+///   row.add(child: .proportional(number, 0.25))
+///   row.add(child: .absolute(status, 50))
+/// }
+/// ```
+///
+/// ## Expected layout:
+///
+/// ┌────────────────────────────┬─┬────────┬─┬──────────────────┐
+/// │            name            │ │ number │ │      status      │
+/// │            (75%)           │ │  (25%) │ │     (50 pts)     │
+/// │                            │ │        │ │                  │
+/// ●──────────── 75  ───────────● ●── 25 ──● ●─────── 50 ───────●
+/// └────────────────────────────┴─┴────────┴─┴──────────────────┘
+/// ●──────────────────────────── 166 ───────────────────────────●
+/// ```
+public struct GridRow: Element {
+    // MARK: - properties -
+    /// The space between children. By default, 0.
+    public var spacing: CGFloat = 0
+
+    /// The child elements to be laid out. By default, an empty array.
+    public var children: [Child] = []
+
+    // MARK: - initialization -
+    public init(configure: (inout GridRow) -> Void) {
```

Comment:
Nit: `= { _ in }` in case someone doesn't want to configure inline for some reason.

Reply:
Fixed.